stixcore/util/scripts/find_unprocessed_tm.py

```diff
@@ -0,0 +1,33 @@
+# helper script to check for unprocessed TM xml files
+# the same logic is implemented into the search_unprocessed_tm_files()
+# method here: stixcore/processing/pipeline.py
+
+import re
+from pprint import pprint
+from pathlib import Path
+
+from stixcore.config.config import CONFIG
+
+unprocessed_tm_files = list()
+TM_REGEX = re.compile(r'.*PktTmRaw.*.xml$')
+logging_dir = Path(CONFIG.get('Pipeline', 'log_dir'))
+logging_dir = Path("/data/stix/out/fits_v1.2.0/logging/")
+tm_dir = Path(CONFIG.get('Paths', 'tm_archive'))
+tm_dir = Path("/data/stix/SOLSOC/from_edds/tm/incomming")
+list_of_log_files = logging_dir.glob('*.xml.out')
+latest_log_file = logging_dir / "TM_BatchRequest.PktTmRaw.SOL.0.2023.305.15.33.13.852.bBWz@2023.352.21.00.01.039.1.xml.out"  # noqa
```

Comment:
hardcoded?

Reply:
it is a helper script for manual work needs updating for dedicated use case anyway